src/ripple/nodestore/tests/util.h

```diff
@@ -0,0 +1,71 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_NODESTORE_TEST_UTIL_H
+#define RIPPLE_NODESTORE_TEST_UTIL_H
+
+#include <ripple/basics/contract.h>
+#include <beast/http/rfc2616.hpp>
+#include <beast/detail/ci_char_traits.hpp>
+#include <boost/regex.hpp>
+#include <map>
+#include <string>
+
+namespace ripple {
+namespace NodeStore {
+
```

Comment:
`parse_args` either needs to be moved to a source, or needs to be given weak linkage (inline or gratuitous template).  I'm getting duplicate symbol linker error without such treatment.